### PR TITLE
added readme markdown

### PR DESCRIPTION
**Pull request by kjknows**

### CODE REVIEW

File: lesson_00/kevinmason/KevREADME.md
Comment (anthonydmays) [inline lesson_00/kevinmason/KevREADME.md:1]: Please fix grammar and capitalization throughout.

Comment (anthonydmays) [inline lesson_00/kevinmason/KevREADME.md:1]: Also, the file name is wrong (should be README.md).

**Review by anthonydmays — CHANGES_REQUESTED** [review]

File: lesson_00/kevinmason/KevREADME.md
Comment (kjknows) [reply lesson_00/kevinmason/KevREADME.md:1]: I fixed the erros

File: lesson_00/kevinmason/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Kevin-s_Read_me
```
Comment (anthonydmays) [inline lesson_00/kevinmason/README.md:1]: Please change the title here (without dashes and underscored).

**Review by anthonydmays — APPROVED** [review]

**Comment by kjknows** [comment]
Changed the title, and took out the dashes

File: lesson_00/kevinmason/README.md (hunk 1 above)
Comment (kjknows) [reply lesson_00/kevinmason/README.md:1]: changed and took dashes out

**Comment by anthonydmays** [comment]
> Changed the title, and took out the dashes

I'm still seeing the dashes and underscores, did you git push?

**Review by anthonydmays — DISMISSED** [review]

**Review by anthonydmays — APPROVED** [review]

**Review by anthonydmays — APPROVED** [review]

**Comment by kjknows** [comment]
I did it straight from get hub.

**Pull request merged**